include/webui.h

```diff
@@ -447,6 +471,28 @@ WEBUI_EXPORT void webui_set_file_handler(size_t window, const void* (*handler)(c
  */
 WEBUI_EXPORT void webui_set_file_handler_window(size_t window, const void* (*handler)(size_t window, const char* filename, int* length));
 
+/**
+ * @brief Set window style. ItShould be used before `webui_show()`.
+ *
+ * @param window The window number
+ * @param id The desired window style from `webui_window_style` enum
+ * @param focused_bgcolor The focused background color (24bit RGB color, default : 0xFFFFFF)
+ * @param unfocused_bgcolor The un-focused background color (24bit RGB color, default : 0xFFFFFF)
+ *
+ * @example webui_set_window_style(myWindow, WEBUI_WINDOW_STYLE_CUSTOM, 0xFFFFFF, 0x808080);
+ */
+WEBUI_EXPORT void webui_set_window_style(size_t window, webui_window_style id, uint32_t focused_bgcolor, uint32_t unfocused_bgcolor);
+
+/**
+ * @brief Start dragging window. (currently windows only)
+ *
+ * @param window The window number
+ *
+ * @return Returns command is successful
+ *
+ * @example webui_start_drag_window(myWindow);
+ */
+WEBUI_EXPORT bool webui_command_window(size_t window, webui_window_command command);
```

Comment:
This does not seem like the right docs for this function.

Reply:
Good I will fix it.